src/guide/api.md

```diff
@@ -366,21 +366,55 @@ HTTP API errors will have the following response body:
 ```
 -->
 
+## `/magic-links/send`
+
+### Request
+
+`POST` requests made to the `/magic-links/send` endpoint emails the address provided with a Magic Link. This magic link contains a URL, provided by you, that will redirect the recipient to an endpoint in your application. From here, you can send the token Passwordless.dev has embedded in the link to verify the token at `signin/verify`.
+
+The request must include all three fields.
+
+- `emailAddress`: Recipient of the magic link. Must be a valid email address.
```

Comment:
Make it clear that the customer or developer integrating magic links is responsible for validating an e-mail address matches the given user id as `Passwordless.dev` does not store e-mails. Otherwise they could introduce a vulnerability on their end. Use a `warning` or `danger` alert box to make it more visible.